src/pseudopeople/interface.py

```diff
@@ -41,7 +43,9 @@ def generate_w2(
     :return: A pd.DataFrame of noised W2 data
     """
     configuration_tree = get_configuration(configuration)
-    data = pd.read_csv(path, dtype=str, keep_default_na=False)
+    data = pd.read_hdf(path)
+    if not isinstance(data, pd.DataFrame):
+        raise TypeError(f"File located at {path} must contain a pandas DataFrame.")
```

Comment:
I'm move into a `load_data` utility function so that we don't forget to check type.